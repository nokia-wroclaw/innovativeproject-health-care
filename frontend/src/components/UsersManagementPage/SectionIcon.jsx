import React from "react";

const SectionIcon = ({ className, onClick, color }) => {
  return (
    <i
      className={className}
      aria-hidden="true"
      style={{ fontSize: "1.3em", cursor: "pointer", marginRight: 8, color }}
      onClick={onClick}
    />
  );
};

export default SectionIcon;
