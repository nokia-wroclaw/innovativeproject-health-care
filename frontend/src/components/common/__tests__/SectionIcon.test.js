import React from "react";
import { shallow } from "enzyme";
import SectionIcon from "../SectionIcon";

describe("SectionIcon", () => {
  it("should match snapshot", () => {
    expect(shallow(<SectionIcon />)).toMatchSnapshot();
  });
});
