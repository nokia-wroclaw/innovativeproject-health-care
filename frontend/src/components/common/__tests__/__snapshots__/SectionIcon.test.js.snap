// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionIcon should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SectionIcon />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "aria-hidden": "true",
      "className": undefined,
      "onClick": undefined,
      "style": Object {
        "color": undefined,
        "cursor": "pointer",
        "fontSize": "1.3em",
        "marginRight": 8,
      },
    },
    "ref": null,
    "rendered": null,
    "type": "i",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "aria-hidden": "true",
        "className": undefined,
        "onClick": undefined,
        "style": Object {
          "color": undefined,
          "cursor": "pointer",
          "fontSize": "1.3em",
          "marginRight": 8,
        },
      },
      "ref": null,
      "rendered": null,
      "type": "i",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
